typescript/sdk/src/token/configUtils.ts

```diff
@@ -109,7 +125,9 @@ export async function expandWarpDeployConfig(
       destinationGas,
       hook: zeroAddress,
       interchainSecurityModule: zeroAddress,
-      proxyAdmin: { owner: config.owner },
+      proxyAdmin: isDeployedAsProxyByChain[chain]
+        ? { owner: config.owner }
```

Comment:
why is this defaulted to the top level owner and not read from the proxyAdmin.owner?

Reply:
Spreading the `config` below will override it if it was defined in the config, but otherwise, the default behaviour is that both the token and the proxy admin have the same owner

Reply:
ah missed that